src/bin/ci/resources/build_application.inc.conf

```diff
@@ -1,3 +1,5 @@
+akka.http.host-connection-pool.max-open-requests = 1024
```

Comment:
Is this related?

Reply:
No but all my PRs keep failing with this `max-open-requests` error so I'm trying to ~~sneak this in~~ fix it